tests/cypress/integration/api/graphqlOrdering.spec.ts

```diff
@@ -0,0 +1,66 @@
+import gql from 'graphql-tag'
+
+describe('Test graphql ordering', () => {
+    before('create a list with 3 children', () => {
+        cy.apollo({
+            mutation: gql`
+                mutation {
+                    jcr(workspace: EDIT) {
+                        addNode(parentPathOrId: "/", name: "testList", primaryNodeType: "jnt:contentList") {
+                            subnode1: addChild(name: "Hello", primaryNodeType: "jnt:bigText") {
+                                uuid
+                            }
+                            subnode2: addChild(name: "Bonjour", primaryNodeType: "jnt:press") {
+                                uuid
+                            }
+                            subnode3: addChild(name: "Hola", primaryNodeType: "jnt:linkList") {
+                                uuid
+                            }
+                        }
+                    }
+                }
+            `,
+        })
+    })
+
+    after('Delete', function () {
```

Comment:
It might be better to detail here what you're deleting. Although this might be useful in the context of that particular suite since you're familiar with it, someone looking at the execution on a CI platform might not know.

For example: `Deleting the list created for executing the tests`